about/people.yml

```diff
@@ -1,177 +1,109 @@
       Bretagne (France) in 2000.
     image: thomas_main.jpg
-  - name: Paul Hall
+  - name: David D Johnson
     type: Full Time
-    team: Advanced Research Computing
-    subteam: High-Performance Computing
-    title: Senior Research Software Engineer
-    github_username: phall-brown
-    brown_directory_uuid: da1c3aed-d801-56a8-0536-0701f6c5c830
-    bio: Paul has a background in computational fluid dynamics and geophysics. He received
-      his Ph.D. in Oceanography from the URI.
-    image: hall_main.jpg
-  - name: Rohit Kakodkar
+    team: CCV Operations
+    title: Senior HPC Systems Engineer
+    github_username: ''
+    brown_directory_uuid: c14eb6ba-4b51-aabd-bf11-ef4071826060
+    bio: ''
+    image: dave_main.jpg
+  - name: Geoffrey Avila
     type: Full Time
-    team: Advanced Research Computing
-    subteam: High-Performance Computing
-    title: Research Software Engineer
-    github_username: rkakodkar
-    brown_directory_uuid: bdc65f4b-8852-4cef-99c3-f765988e15c0
-    bio: Rohit completed his Ph.D. in Mechanical Engineering at the University of Delaware, 
-      where his research involved developing a large-scale simulation code base to calculate 
-      thermal properties of nanostructured materials. Using these simulations, he worked on developing 
-      materials that would be more efficient at thermal management inside microprocessors.
-    image: rohit_main.jpg
-  - name: Ashok Ragavendran
+    team: CCV Operations
+    title: Senior HPC Systems Engineer
+    github_username: ''
+    brown_directory_uuid: f904bb23-202f-48fb-bf93-7b7d326c7043
+    image: avila_main.jpg
+  - name:  Nathan Wood
     type: Full Time
-    team: Advanced Research Computing
-    subteam: Computational Biology Core
-    title: Lead Data Scientist
-    github_username: ashokrags
-    brown_directory_uuid: 60802d0b-8f18-499a-a1f3-5f9b24c390e4
-    bio: Ashok is a computational biologist/data scientist with a varied background
-      in genetics, genomics, statistics and computation. He has a PhD in Fisheries and
-      Wildlife from MSU and has worked in areas of quantitave genetics, biomedical genetics,
-      ecology and bioinformatics. Above all he has a keen interest in data analysis,
-      computing and providing user friendly solutions for analysis and visualization
-      of large data sets. Otherwise he keeps busy with two boys, two pups and life near
-      the beach.
-    image: ashok_main.jpg
-  - name: Mete Tunca
+    team: CCV Operations
+    title: Senior Research Systems Administrator
+    github_username: n8wood-brown
+    brown_directory_uuid: 857b928c-2ce1-fb2a-cde7-1a4ff21fa2d3
+    bio:
+    image: nate_main.jpg
+  - name: Paulo S Baptista
     type: Full Time
-    team: Leadership
-    title: Assistant Director of Cloud and Research Computing
+    team: CCV Operations
+    title: Senior Research Systems Administrator
     github_username: ''
```

Comment:
@paulobaptista 's github username should be added here

Reply:
Fixed !